<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vmessOutEditor</class>
 <widget class="QWidget" name="vmessOutEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>UUID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="idLineEdit">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>36</number>
     </property>
     <property name="placeholderText">
      <string notr="true">xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="alterLabel">
     <property name="text">
      <string>Alter ID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="alterLineEdit">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="securityLabel">
     <property name="text">
      <string>Security</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="securityCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string notr="true">auto</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">aes-128-gcm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">chacha20-poly1305</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">none</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
